cli/azd/pkg/infra/azure_resource_types.go

```diff
@@ -72,6 +84,17 @@ func IsTopLevelResourceType(resourceType AzureResourceType) bool {
 		return false
 	}
 
+	// Temporal patch displaying Azure SQL and Azure Mongo instead of only Azure Cosmos DB
+	// Both resources uses the same top-level grouper Microsoft.DocumentDB/databaseAccounts
+	// We need a more general solution in the future for naming resources
+	// This patch let the known-resource-type name to be used for static translation
+	if resourceType == AzureResourceTypeCosmosSQL || resourceType == AzureResourceTypeCosmosMongo {
+		return true
+	}
+	if resourceType == AzureResourceTypeCosmosDb {
+		return false // So we don't try to translate the grouping resource for mongo or SQL when using Cosmos
+	}
```

Comment:
For the bug fix I don't think we need this at the moment since it was referring to standard SQL for Azure (SQL server / db), not the cosmos flavor.  

When referring to cosmos, it would be nice to see the chosen API, but I think we get this for free now after defining the sub resource type.

Reply:
Yeah, I will close this PR.  
I'd wait for now on making a difference between cosmos APIs